core/dynamic-tags/tag.php

```diff
@@ -51,7 +52,7 @@ public function get_content( array $options = [] ) {
 			endif;
 
 		} elseif ( ! Utils::is_empty( $settings, 'fallback' ) ) {
-			$value = $settings['fallback'];
+			$value = Str::sanitize_input_string_or_array( $settings['fallback'] );
```

Comment:
why not `wp_kses_post` like others?

```suggestion
			$value = wp_kses_post( $settings['fallback'] );
```

Reply:
@StyleShit `Str::sanitize_input_string_or_array()` will just be a wrapper for `wp_kses_post()` to handle scenarios where the fallback value might be an array.

Reply:
@StyleShit Checking...

Reply:
@StyleShit When the fallback is a file upload for example.

Reply:
@louiswol94 `wp_kses_post_deep()` looks equivalent to `sanitize_input_string_or_array()`. I like your approach because we can extend the function to handle future unexpected changes to ACF data types if needed as long as the function signature (naming in this case) does not make extension unexpected.

@StyleShit we reverted this change in a previous release because image dynamic tags sometimes returns an array. This was unexpected when I tested for the release.

Reply:
@ArmanPurtoian Looks like `wp_kses_post_deep()` should work. Can you please implement? You'll also have to remove your unit tests from the `test-str.php` file and see if there's another relevant way to test this - like for example, creating a new tag and testing the output of `get_content()`.

Thanks @StyleShit

Reply:
> @louiswol94 `wp_kses_post_deep()` looks equivalent to `sanitize_input_string_or_array()`. I like your approach because we can extend the function to handle future unexpected changes to ACF data types if needed as long as the function signature (naming in this case) does not make extension unexpected.
> 
> @StyleShit we reverted this change in a previous release because image dynamic tags sometimes returns an array. This was unexpected when I tested for the release.

Thanks Barco. I prefer to rather use built-in WP functions if we can 😄

Reply:
@StyleShit @ArmanPurtoian @Barcov I am going to approve this PR. We don't have to test a WP function `wp_kses_post_deep()`, and if we want to further test the tag values we can rather expand Pro tests such as `test-acf-date-time.php`.

Reply:
@louiswol94 

what about testing the tag itself?